src/App.css

```diff
@@ -1,39 +1,33 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.calcBody {
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  margin-top: 10%;
+}
+
+.inputUser {
+  background-color: #858694;
+  color: #efeff1;
+  font-weight: bold;
+  border: none;
+  width: 43.4vw;
+  height: 5vw;
+  font-size: larger;
+  direction: rtl;
 }
 
-.App-link {
-  color: #61dafb;
+.backClr {
+  background-color: #e0e0e0;
+  border: 1px solid #cdcdcd;
+  width: 11vw;
+  height: 10vw;
+  font-weight: bold;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+.zero {
+  width: 22vw;
+}
```

Comment:
Look at the image below:
- [ ] Despite zooming into about 25%, your calculator still looks unnecessarily large. Instead of `vw` and `vh`, I would suggest using `rem` for your width and height calibrations so that your app design looks nicer. For example:
#### NOTE: You can use other values, these are merely suggested values.
```
.inputUser {
  ...
  width: 19.8rem;
  height: 3rem;
  ....
}

.backClr {
  ...
  width: 5rem;
  height: 5rem;
  ...
}

.zero {
  width: 10rem;
}
```

![atilla-ui](https://user-images.githubusercontent.com/36542986/144444218-16c0c39e-312b-43c0-8242-66af4177a49f.jpg)

Reply:
Yeah, you're welcome